enum.dd

```diff
@@ -260,7 +260,23 @@ enum size = __traits(classInstanceSize, Foo);  // evaluated at compile-time
 	$(P Using manifest constants is an idiomatic D method
 	to force compile-time evaluation of an expression.)
 
-
+        ---
+        template Foo(T)
+        {
+            // Not bad, but the 'size' variable will be located in the executable.
```

Comment:
At least in object code. If you take its address (auto p = &Foo!T.size), it would also go into exe file.